### DISCUSSION TITLE
A way to copy commands from gui or load through cmd line profile

### DISCUSSION THREAD

**Discussion by stl3** [LostRuins/koboldcpp#850] [Ideas]
I was hoping for an easier method to copy options loaded from gui when going to command line, as it is a little tedious to go through every single custom option, so I was wondering if it's possible to integrate a copy/paste button from gui to copy the cmd line options to clipboard, or have it gen the options you used to a file like `koboldcpp --profile example.kcpps --cmds cmds.txt` and then manually copy paste the commands from the cmd.txt for example. I understand this may be something niche, but could also be useful for those who go between gui/cmd line a lot. Going from cmd line to gui is less painful though because you don't have to type the text all out, just the other way around is.

**Comment by LostRuins** [comment]
The kcpps file should contain the same args as what you need for the command line already, the formatting is just a little different

**Reply by stl3** [reply]
It didn't occur to me to look inside the file, my brain is primed to automatically expect some kind of binary file haha. I think I'll manage by making a parser to simplify the process. Thanks!
